server/routes/barcode/image/GenerateBarcodeImageController.test.ts

```diff
@@ -70,6 +70,12 @@ describe('GenerateBarcodeImageController', () => {
               barcodeImageUrl: 'barcode-address-image-url',
               barcodeImageName: `SendLegalMail-John-Smith-${moment().format('YYYY-MM-DD')}.png`,
               recipientName: 'John Smith',
+              copyButtonHtml: `Copy <span class="govuk-visually-hidden"> barcode image for John Smith (copies PNG file SendLegalMail-John-Smith-${moment().format(
+                'YYYY-MM-DD'
+              )}.png of approximate size 0.2 Megabytes to the Clipboard)</span>`,
+              downloadButtonHtml: `Download <span class="govuk-visually-hidden"> barcode image for John Smith (downloads PNG file SendLegalMail-John-Smith-${moment().format(
+                'YYYY-MM-DD'
+              )}.png of approximate size 0.2 Megabytes)</span>`,
```

Comment:
All file downloads need to specify the file type and size for users of screen readers.